### PR TITLE
Harry palz bio

### PR DESCRIPTION
**Pull request by HarryPalz**
This is my bio for @HarryPalz as requested in #5 .
:kissing_cat:

### CODE REVIEW

File: docs/HarryPalz.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#HarryPalz Bio
+###An autobiography by HarryPalz
+
+**Name** Harry Palzewicz
+
+**Location** `docs` folder
```
Comment (pdgessler) [inline docs/HarryPalz.md]: It asked for hometown, man! Not location!!! :eyes:

**Comment by musicfuel** [comment]
I'd think about removing the Location and replacing that with Hometown and Occupation per the requirements.

**Comment by JoshuaSCochrane** [comment]
Needs moar licecap. Like way more.

File: docs/HarryPalz.md (hunk 1 above)
Comment (HarryPalz) [inline docs/HarryPalz.md]: OH GOD IM SO CONFUSED

**Comment by musicfuel** [comment]
@JoshuaSCochrane totally. I can't believe I missed that :disappointed:

**Comment by HarryPalz** [comment]
@JoshuaSCochrane @musicfuel 
It's liceCAP.

@musicfuel 
I can fix this, but you were supposed to review with inline comments per the requirements!

**Comment by HarryPalz** [comment]
:shipit: 
Oh wait, @pdgessler already did that...

**Comment by pdgessler** [comment]
@HarryPalz tit for tat

**Comment by pdgessler** [comment]
I can do this _allll_ day....

**Comment by HarryPalz** [comment]
Actually, #5 is closed meow

**Pull request merged**